lib/components/template/layer/LayerManager.vue

```diff
@@ -120,6 +125,14 @@ v-layout(column)
         this.layerToDelete = layer
       },
 
+      setLayerToggling(layer) {
+        this.toggleLayer(layer)
```

Comment:
Don't need to wrap the action invocation like this, can just call to `toggleLayer(layer)` in the template directly.

Reply:
Good to know! I'm still a bit confused on the template part of all of this.